code/modules/vehicles/mecha/mecha_actions.dm

```diff
@@ -216,13 +224,13 @@
 	var/new_damtype
 	switch(chassis.damtype)
 		if("tox")
-			new_damtype = "brute"
+			new_damtype = BRUTE
 			to_chat(owner, "[icon2html(chassis, owner)]<span class='notice'>Your exosuit's hands form into fists.</span>")
 		if("brute")
-			new_damtype = "fire"
+			new_damtype = BURN
 			to_chat(owner, "[icon2html(chassis, owner)]<span class='notice'>A torch tip extends from your exosuit's hand, glowing red.</span>")
 		if("fire")
```

Comment:
you set them to defines but you dont check for those defines here

Reply:
Good point